common/hashing.h

```diff
@@ -49,7 +49,7 @@ class HashCode : public Printable<HashCode> {
   // useful when using the hash code to index a hash table. It prioritizes
   // computing the index from the bits in the hash code with the highest
   // entropy.
-  constexpr auto ExtractIndex(ssize_t size) -> ssize_t;
+  constexpr auto ExtractIndex() -> ssize_t;
```

Comment:
Update doc comment to reflect this change

Reply:
Done, thanks!